test/benchmarks/bench_classify.rb

```diff
@@ -40,7 +42,7 @@ def bench_allocations_without_cache
     Primer::Classify::Cache.clear!
     Primer::Classify.call(**@values)
 
-    assert_allocations 68 do
+    assert_allocations 69 do
```

Comment:
Interesting. Any ideas why these allocations increased?

Reply:
I'm not sure! I ran the benchmarks on ruby 2.7 with the rubocop fixes and they passed, which indicates to me the extra allocations were introduced in ruby 3.0. The numbers also aren't particularly consistent between test runs, but I'm not sure if that's because of ruby 3 or if the benchmarks are inherently "noisy.: